charts/fake-aws-s3/values.yaml

```diff
@@ -15,5 +15,3 @@ minio:
     - name: public
       purge: false
       policy: public
-  environment:
-    MINIO_BROWSER: "off"
```

Comment:
is this not needed anymore in the new chart (previously is loaded a publicly-accessible graphical user interface to poke around all the contents in minio with insecure credentials)

Reply:
It's not configurable in the new chart. At least; it complained when I had it set about the `environment` option for some reason.

It's as publicly accessible as the S3 API (it's hosted exposed on the same Kubernetes service).  I don't see more or less risk in enabling or disabling it.   Both the S3 API and the GUI use the same  AWS credentials.  If you use weak credentials then yes this is problematic. But this is orthogonal to the GUI. If exposing the Web UI is a problem then so is the API in general.

I actually found it useful for debugging to be able to access the API graphically.